### PR TITLE
fix headers key

### LINKED ISSUE #3: Need to change the key for response headers
I guess a [change](https://github.com/pinojs/pino/pull/529/files) in how the response headers are written in Pino unfortunately breaks this module.

### PR DESCRIPTION
**Pull request by henriklundgren**
This is a breaking change.

1. Pino(-http) unfortunately doesn't seem to expose httpVersion so I removed that.
  Maybe open PR in Pino(-http) to add it, and wait with merging this change?
2. This is breaking but we could maybe check for existence of header or headers key if needed!?
3. One of the cli tests was being difficult with me :(

closes #3

### CODE REVIEW

File: test/cmd.js [hunk 1]
```diff
@@ -269,8 +269,6 @@ test('-k -a 2', function (t) {
   })
 
   t.is(result.output[1] + '', expected)
-  t.is(result.output[2] + '', log + msg)
```
Comment (mcollina) [inline test/cmd.js]: why was this removed?

Comment (henriklundgren) [reply test/cmd.js]: Fixed, sorry about that :)

**Comment by henriklundgren** [comment]
I don't think any new tests should be needed for this change. They are already in place from before.

**Comment by andyli** [comment]
I've just tried this out since the current version is indeed broken with the latest express-pino-logger.
However, this PR didn't work for me either:

```
/workspace/node_modules/pino-toke/tokens.js:70
    return key.trim()
               ^

TypeError: key.trim is not a function
    at Object.res (/workspace/node_modules/pino-toke/tokens.js:70:16)
    at eval (eval at compile (/workspace/node_modules/pino-toke/index.js:37:10), <anonymous>:3:135)
    at DestroyableTransform._transform (/workspace/node_modules/pino-toke/index.js:26:14)
    at DestroyableTransform.Transform._read (/workspace/node_modules/readable-stream/lib/_stream_transform.js:184:10)
    at DestroyableTransform.Transform._write (/workspace/node_modules/readable-stream/lib/_stream_transform.js:172:83)
    at doWrite (/workspace/node_modules/readable-stream/lib/_stream_writable.js:428:64)
    at writeOrBuffer (/workspace/node_modules/readable-stream/lib/_stream_writable.js:417:5)
    at DestroyableTransform.Writable.write (/workspace/node_modules/readable-stream/lib/_stream_writable.js:334:11)
    at DestroyableTransform.ondata (/workspace/node_modules/readable-stream/lib/_stream_readable.js:619:20)
    at emitOne (events.js:116:13)
Error: write EPIPE
    at WriteWrap.afterWrite (net.js:868:14)
```

I can see that it's because `res.headers["content-length"]` is a number, not a string, so there is no `trim()`.

**Comment by jonfrost43** [comment]
@henriklundgren is there any need to trim the header value? 

Since the values are coming from the headers object now, perhaps they can returned without mutation.

Alternatively, ensure the type of `key` is String before calling `.trim()`

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   "license": "MIT",
   "devDependencies": {
     "standard": "^7.1.2",
-    "tap": "^12.0.1"
+    "tap": "12.1.1"
```
Comment (mcollina) [inline package.json]: Can you please restore the `^`?

Comment (henriklundgren) [reply package.json]: I remember this being something about Pino supporting node version >=4 and tap moved to >=8.

Is this something you can confirm?

Comment (mcollina) [reply package.json]: Confirmed, but `^` does not include semver-major changes.

Comment (henriklundgren) [reply package.json]: Reverted to `^12.0.1`.

**Comment by henriklundgren** [comment]
@mcollina I removed http version from the readme. Is this enough?

**Comment by Dinoshauer** [comment]
Would it be possible to get this merged and released?
`pino-toke` currently is broken for me, I had to install it with `yarn add pinojs/pino-toke#3871dd4`

**Pull request merged**